waspc/src/Wasp/SemanticVersion.hs

```diff
@@ -19,11 +19,12 @@ import Text.Printf (printf)
 
 -- Implements SemVer (semantic versioning) by following spec from https://github.com/npm/node-semver .
 
-data Version = Version
-  { major :: Natural,
-    minor :: Natural,
-    patch :: Natural
-  }
+data Version
+  = Version
```

Comment:
Can we format to be as it was?

Reply:
Yep, it's failing the CI, I had to turn of autoformat for a second (package.json get messed up otherwise)